src/routes/SubmissionRoutes.js

```diff
@@ -8,33 +8,38 @@ module.exports = {
       controller: 'SubmissionController',
       method: 'createSubmission',
       auth: 'jwt',
-      access: ['Topcoder User', 'Administrator', 'Copilot']
+      access: ['Topcoder User', 'Administrator', 'Copilot'],
+      scopes: ['write:submission', 'all:submission']
     }
   },
   '/submissions/:submissionId': {
     get: {
       controller: 'SubmissionController',
       method: 'getSubmission',
       auth: 'jwt',
-      access: ['Topcoder User', 'Administrator', 'Copilot']
+      access: ['Topcoder User', 'Administrator', 'Copilot'],
+      scopes: ['read:submission', 'all:submission']
     },
     put: {
       controller: 'SubmissionController',
       method: 'updateSubmission',
       auth: 'jwt',
-      access: ['Administrator']
+      access: ['Administrator'],
+      scopes: ['write:submission', 'all:submission']
     },
     patch: {
       controller: 'SubmissionController',
       method: 'patchSubmission',
       auth: 'jwt',
-      access: ['Administrator']
+      access: ['Administrator'],
+      scopes: ['write:submission', 'all:submission']
```

Comment:
wondering how we restrict users to PATCHing their own submissions and then limiting admins to PATCHing anyones submission. Perhaps I should add another scope for this? 
@mtwomey for thoughts